tests/data/external_account_authorized_user_non_gdu.json

```diff
@@ -0,0 +1,10 @@
+{
+    "type": "external_account_authorized_user",
+    "audience": "//iam.googleapis.com/locations/global/workforcePools/$WORKFORCE_POOL_ID/providers/$PROVIDER_ID",
```

Comment:
endpoints should should not be pointing to googleapis, but instead the value of universe_domain 

also missing new line eof

Reply:
updated